stdlib/_typeshed/__init__.pyi

```diff
@@ -69,9 +69,12 @@ SupportsRichComparisonT = TypeVar("SupportsRichComparisonT", bound=SupportsRichC
 
 # Dunder protocols
 
-class SupportsAdd(Protocol):
+class SupportsAddition(Protocol):
```

Comment:
(I think I prefer the old name since it mirrors the dunder method name)

Reply:
I disliked the idea of `SupportsAddWithInt` ://

But fair enough, I'll revert the naming change :)